frontend/src/pulldasher/notifications.tsx

```diff
@@ -0,0 +1,18 @@
+import { useNotification } from "../notifications";
+import { getUser } from "../page-context";
+import { Pull } from "../pull";
+
+export function usePullNotification(pulls: Pull[], action: string) {
+  useNotification<Pull>(pulls, {
+    key: (pull: Pull) => pull.number,
+    filter: (pull: Pull) => pull.isMine() || pull.hasOutdatedSig(getUser()),
+    message(pulls) {
+      const titles = pulls.map((p) => p.title).join(", ");
+      console.log(action, titles);
```

Comment:
```suggestion
      const titles = pulls.map((p) => p.title).join(", ");
      new Audio('https://audio-previews.elements.envatousercontent.com/files/105394911/preview.mp3?response-content-disposition=attachment%3B+filename%3D%22QKTA234-pop.mp3%22').play()
```

https://mixkit.co/free-sound-effects/pop/

Reply:
[ad9d045](https://github.com/iFixit/pulldasher/pull/327/commits/ad9d045242d11c28493d84f37832eb2aea74dfe7)